### PR TITLE
VR-3805 visibility implementation

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java [hunk 1]
```diff
@@ -448,7 +448,12 @@ public static Throwable findRootCause(Throwable throwable) {
                 .addDetails(Any.pack(defaultInstance))
                 .build();
       } else {
-        LOGGER.error("Exception occurred:", e);
+        LOGGER.error("Exception occurred: {}", e.getMessage());
+        StackTraceElement[] stack = e.getStackTrace();
```
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java:452]: what motivated this?

**Review by ravishetye — APPROVED** [review]

File: backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java (hunk 1 above)
Comment (zlv) [reply backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java:452]: When there is some bug, I can't see why this bug happened in Kibana without printing stack trace. I can just remove this code from here and do this stacktrace print in separate PR or use some log4j tools for that

Comment (evgeniilezhnin) [reply backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java:452]: When there is some bug, I can't see why this bug happened in Kibana without printing stack trace. I can just remove this code from here and do this stacktrace print in separate PR or use some log4j tools for that

Comment (ravishetye) [reply backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java:452]: no this is fine for now.
in future either bubble this up in PR description or adda PR comment as a self review

**Pull request merged**